### DISCUSSION TITLE
What target to use for Diatone Mamba Stack - MK4 F405

### DISCUSSION THREAD

**Discussion by firozdang** [iNavFlight/inav#8399] [Q&A] [answered]
I bought Diatone Mamba Stack - MK4 F405, inav configurator is a bit confusing, there are MAMBAF405, F405 2022A, F405 2022B, and F405 US and F405 I2c, where can I find the information which target belongs to which processor? 

https://www.getfpv.com/electronics/stacks/diatone-mamba-stack-mk4-f722-fc-40a-3-6s-blheli-32-128k-esc-mini-20x20.html

**Comment by DzikuVx** [comment, marked as answer]
Please contact Diatone customer support with this question. But target name should be written on the label of the FC packaging

**Reply by firozdang** [reply]
Thanks Pawel, will check out
